code/modules/clothing/head/jobs.dm

```diff
@@ -787,12 +787,14 @@
 /obj/item/clothing/head/beret/centcom_formal
 	name = "\improper CentCom Formal Beret"
 	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Nanotrasen's most recent nano-fabric durability enhancements, this time, it's not the case."
-	icon_state = "beret_badge"
-	greyscale_config = /datum/greyscale_config/beret_badge
-	greyscale_config_worn = /datum/greyscale_config/beret_badge/worn
-	greyscale_colors = "#46b946#f2c42e"
+	icon_state = "centberet" // Bubberstation Edit
```

Comment:
You can easily make a lot of the clothing stuff modular just by redefining it in our modular files.

Reply:
> You can easily make a lot of the clothing stuff modular just by redefining it in our modular files.

Fair enough, working on it.